src/Type/ApiTypes.ts

```diff
@@ -322,7 +322,7 @@ export interface WorkOrderResponse {
   }
 }
 
-export interface MetaDataResponse {
+export type MetaDataResponse = {

```

Comment:
don't change interface to type.
This is what i meant.
![image](https://github.com/Sunaina04/Marina-Mooring/assets/147302611/ef725019-ab94-439b-afe6-36338471167b)